### PR TITLE
Web enhancement features

### PR DESCRIPTION
**Pull request by KambleSonam**
1. Increment decrement user profile properties
2. Multi Value user profile properties
3. dismissSpamControl for popup and exit intent
4. setLogLevel logic fix

### CODE REVIEW

File: example-apps/react/src/index.js [hunk 1]
```diff
@@ -5,8 +5,8 @@ import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
 
-clevertap.init('YOUR_ACCOUNT_ID'); // Replace YOUR_ACCOUNT_ID, can be initialized just once
-
+clevertap.init('your account id'); // Replace YOUR_ACCOUNT_ID, can be initialized just once
```
Comment (akashvercetti) [inline example-apps/react/src/index.js:8]: optional: the comment can be worded a bit better

File: src/clevertap.js [hunk 2]
```diff
@@ -41,6 +48,7 @@ export default class CleverTap {
   #isSpa
   #previousUrl
   #boundCheckPageChanged = this.#checkPageChanged.bind(this)
+  #isDisableWebPopUpSpamControl
```
Comment (akashvercetti) [inline src/clevertap.js:51]: optional: can `isDisableWebPopUpSpamControl` be better named as `isWebPopUpSpamControlDisabled` ?

File: src/clevertap.js [hunk 3]
```diff
@@ -176,6 +194,66 @@ export default class CleverTap {
...
 
+    // Method to get location - lat, long
+    /**
+     *
+     * @param {number} lat
+     * @param {number} lng
+     * @param {callback function} handleCoordinates
+     * @returns
+     */
+    // this.getLocation = function (lat, lng) {
```
Comment (akashvercetti) [inline src/clevertap.js:205]: can the commented methods be removed?

File: src/clevertap.js [hunk 4]
```diff
@@ -209,6 +287,66 @@ export default class CleverTap {
...
+
+    this.handleDecrementValue = (key, value) => {
+      this.profile._handleIncrementDecrementValue(key, value, COMMAND_DECREMENT)
+    }
+
+    this.setMultiValuesForKey = (key, value) => {
+      if (Array.isArray(value)) {
+        this.profile._handleMultiValueSet(key, value, COMMAND_SET)
+      } else {
+        console.log('Value should be of type array.')
```
Comment (akashvercetti) [inline src/clevertap.js:252]: optional: the error message can be more descriptive like "setMultiValuesForKey should be called with a value of type array". and maybe console.error() to make it more prominent red.

File: src/clevertap.js (hunk 2 above)
Comment (KambleSonam) [reply src/clevertap.js]: updated to isWebPopUpSpamControlDisabled

File: src/clevertap.js (hunk 3 above)
Comment (KambleSonam) [reply src/clevertap.js]: Removed comments

File: src/clevertap.js (hunk 4 above)
Comment (KambleSonam) [reply src/clevertap.js]: Error message corrected for all methods

File: src/clevertap.js [hunk 5]
```diff
@@ -454,4 +593,35 @@ export default class CleverTap {
...
+      profileObj.tz = new Date().toString().match(/([A-Z]+[\+-][0-9]+)/)[1]
+    }
+    data.profile = profileObj
+    if (payload) {
+      const keys = Object.keys(payload)
+      keys.forEach(key => {
+        data.af[key] = payload[key]
+      })
+    }
+    console.log({ data })
```
Comment (akashvercetti) [inline src/clevertap.js:557]: optional: maybe this log is for debug only

**Review by akashvercetti — APPROVED** [review]

**Pull request merged**